code/game/objects/items/devices/flash.dm

```diff
@@ -93,14 +101,15 @@
 			if(M.flash_eyes(1, 1))
 				M.AdjustConfused(power)
 				terrible_conversion_proc(M, user)
-				M.Stun(2 SECONDS)
-				visible_message("<span class='disarm'>[user] blinds [M] with the flash!</span>")
-				to_chat(user, "<span class='danger'>You blind [M] with the flash!</span>")
-				to_chat(M, "<span class='userdanger'>[user] blinds you with the flash!</span>")
+				M.drop_l_hand()
+				M.drop_r_hand()
+				visible_message("<span class='disarm'>[user] blinds [M] with the [name]!</span>")
+				to_chat(user, "<span class='danger'>You blind [M] with the [name]!</span>")
+				to_chat(M, "<span class='userdanger'>[user] blinds you with the [name]!</span>")
 			else
-				visible_message("<span class='disarm'>[user] fails to blind [M] with the flash!</span>")
-				to_chat(user, "<span class='warning'>You fail to blind [M] with the flash!</span>")
-				to_chat(M, "<span class='danger'>[user] fails to blind you with the flash!</span>")
+				visible_message("<span class='disarm'>[user] fails to blind [M] with the [name]!</span>")
+				to_chat(user, "<span class='warning'>You fail to blind [M] with the [name]!</span>")
+				to_chat(M, "<span class='danger'>[user] fails to blind you with the [name]!</span>")
```

Comment:
You should use `[src]` here ,not `the [name]`. `[src]` automatically adds `the` and is preferred.
Use `[name]` only when you are trying to work around that extra `the` (like below)

See https://github.com/ParadiseSS13/Paradise/blob/master/.github/CONTRIBUTING.md#do-not-use-the

Reply:
Got it. Will fixos asap.